src/A2C_model.py~

```diff
@@ -13,25 +13,30 @@ class ProbabilityDistribution(Model):
 class Model(Model):
     def __init__(self, ac_dim):
         super(Model, self).__init__()
-        self.conv1 = Conv2D(16, 3, activation='sigmoid', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv2 = Conv2D(16, 3, activation='sigmoid', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv3 = Conv2D(32, 3, activation='sigmoid', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv1 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv2 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
```

Comment:
Maybe even pass the activation as an argument (maybe have a dictionary of kwargs if there are a lot of arguments)